src/views/ApartmentApp/components/StudentApplication/components/OffCampusSection/components/OffCampusListItem/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
Do we want this file? It is odd that it is in develop but not apartapp-develop

Reply:
That is the file nick made. His branch was merged into this branch, which in turn is going into apartapp-develop.
So it did not come from develop, it came from apartapp-DevelopOffCampus